crud_file.c

```diff
@@ -0,0 +1,217 @@
+#include <stdio.h>
```

Comment:
1. While running, Input process is not working properly(Asking for ID and name at the same time). Correct it

![image](https://github.com/user-attachments/assets/04319815-30f3-4417-980c-3f4c8f666993)

2.  User data is not displaying at all

![image](https://github.com/user-attachments/assets/25c3241d-dbee-482e-b7ec-cdf3954fcec7)

**It's not in the running state. Make it work to do all possible types of functions including Displaying users, deleting them, etc.**